### PR TITLE
Make default file recording directory of BrowserWebDriverContainer platform independent

### PR DESCRIPTION
**Pull request by kiview**
Fixes incompatibility of Windows with default recording directory, makes Windows work OOTB.

Not 100% sure how to best handle the `IOException` in the constructor.

### CODE REVIEW

File: modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java [hunk 1]
```diff
@@ -79,6 +82,14 @@ public BrowserWebDriverContainer() {
                 .withStrategy(new HostPortWaitStrategy())
                 .withStartupTimeout(Duration.of(15, SECONDS));
 
+        try {
```
Comment (bsideup) [inline modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java:85]: IMO we should move it to `configure` and create the dir only if recording is enabled

Comment (kiview) [reply modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java]: Moved the code.

File: modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java [hunk 2]
```diff
@@ -52,6 +54,7 @@
     private static final int VNC_PORT = 5900;
 
     private static final String NO_PROXY_KEY = "no_proxy";
+    public static final String TC_TEMP_DIR_PREFIX = "tc";
```
Comment (bsideup) [inline modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java:57]: Private?

Comment (kiview) [reply modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java]: Of course, sorry.

**Comment by rnorth** [comment]
This was released in https://github.com/testcontainers/testcontainers-java/releases/tag/1.14.0 🎉

Thanks for the contribution!

**Pull request merged**